bagel/cli.py

```diff
@@ -95,6 +96,26 @@ def help_option():
     )
 
 
+def show_version(value: bool):
+    """Callback to show package version number and exit."""
+    if value:
+        print(f"bagel {__version__}")
+        raise typer.Exit()
```

Comment:
Out of interest / to increase bus factor: could you explain why this is a separate function and cannot be part of the callback function itself?